.github/workflows/prod-deployment.yaml

```
@@ -0,0 +1,82 @@
+env:
+  ECR_REPOSITORY: ${{ secrets.ECR_REPOSITORY_AL }}
+  EKS_CLUSTER_NAME: ${{ secrets.EKS_CLUSTER_NAME_PROD_AL }}
+  AWS_REGION: ${{ secrets.AWS_REGION_NAME_AL }}
+
+jobs:
+  LMS-TAG-BASE-DEPLOYMENT-PROD:
+    name: Deployment
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)